### DISCUSSION TITLE
Can I get moor documentation for isolates (Not Drift)

### DISCUSSION THREAD

**Discussion by J-Nokwal** [simolus3/drift#1574] [Q&A]
Can I get moor documentation for isolates (Not Drift) 
How to implement it?

**Comment by westito** [comment]
Drift is the same as Moor, just renamed
Documentation is here: https://drift.simonbinder.eu/docs/advanced-features/isolates/

**Reply by simolus3** [reply]
Yes, the only difference is that you'll need to import `package:moor/isolate.dart` instead of `package:drift/isolate.dart`.
